src/html/blocks/menu/project-list-item.html

```diff
@@ -0,0 +1,7 @@
+<a href="@@href" class="list-item project-item">
+    <span class="project-item__marker color_@@color"></span>
+    <span class="project-item__title"> @@project </span>
+    <div class="project-item__count">
+        <span class="badge badge_circle color_@@color">@@count</span>
+    </div>
+</a>
```

Comment:
Will be better use one modifier for changing project colors. project-item_@@color
<span class="project-item marker color@@color"></span> - this block can be created by :before